commands/wallet.js

```diff
@@ -37,8 +37,24 @@ const wallet = (reply, message) => {
     })
 }
 
-module.exports.init = controller => controller.hears(
-    ['!wallet (.*)'],
-    'ambient,bot_message,direct_message,direct_mention,mention',
-    (bot, message) => wallet(bot.reply, message)
-)
+module.exports.init = (controller, general) => {
+    controller.hears(
+        ['!wallet (.*)'],
+        'bot_message',
+        (bot, message) => message.channel == general
```

Comment:
Nit-Pick, but I would use brackets if it's more than one like.

Reply:
I'm not following?